framework/vulkan_sample.cpp

```diff
@@ -190,16 +190,19 @@ void VulkanSample::update(float delta_time)
 {
 	update_scene(delta_time);
 
-	update_stats(delta_time);
-
 	update_gui(delta_time);
 
 	auto &command_buffer = render_context->begin();
 
+	uint32_t active_frame_idx = render_context->get_active_frame_index();
```

Comment:
Perhaps add a comment here explaining that this section collects the performance data for the sample graphs, from todays meeting I've been reminded how vital this part is to the whole project and I think having some extra stress here would be nice

Reply:
Done